Project/searchForm.js

```diff
@@ -0,0 +1,110 @@
+
+
+
+class CompanyInfo {
```

Comment:
if you have a file that contains only one class, the convention is to have the same name for both of them